3-enrich/stream-enrich/src/main/scala/com.snowplowanalytics.snowplow.enrich.stream/sources/AbstractSource.scala

```diff
@@ -154,15 +157,20 @@ abstract class AbstractSource(
 
   implicit val resolver: Resolver = igluResolver
 
+  def getPiiEvent(event: EnrichedEvent): Option[String] = {
+    if (event.pii == null || event.pii.isEmpty) None
```

Comment:
So, it wasn't changed in the end?

Reply:
This was changed: https://github.com/snowplow/snowplow/pull/3646/files#diff-e405d498f6ff411d9e07f91778055e48R173 I think you are looking at an outdated version..